.codeclimate.yml

```diff
@@ -1,7 +1,17 @@
+version: "2"
+checks:
+  complex-logic:
+    enabled: false
```

Comment:
Idk. I kinda feel these are valuable.

Reply:
I think they are but I'm no expert in codeclimatology -- How do you instruct it to ignore single files or methods?